js/quiz.js

```diff
@@ -176,4 +282,5 @@ function selectDifficulty(grade) {
   for (var i = 0; i < items2.length; i++) {
     items2[i].addEventListener("click", function() { nextQuestion(this, 'fu'); }, false);
   };
-};
+
+};
```

Comment:
```suggestion
};

```
we need a newline at the end of the file